scripts/commands/updateApiDocs.ts

```diff
@@ -206,14 +250,19 @@ async function convertHtmlToMarkdown(
   baseSourceUrl: string,
   pkg: Pkg,
   version: string,
+  legacyReleaseNoteEntries: { title: string; url: string }[],
 ) {
-  const globs = ["apidocs/**.html", "apidoc/**.html", "stubs/**.html"];
-  if (pkg.name !== "qiskit") {
-    globs.push("release_notes.html");
-  }
-  const files = await globby(globs, {
-    cwd: htmlPath,
-  });
+  const files = await globby(
+    [
+      "apidocs/**.html",
+      "apidoc/**.html",
+      "stubs/**.html",
+      "release_notes.html",
```

Comment:
Wait why is this change now safe to make? I thought these legacy release notes aren't meant to be added via this script yet? That you did it manually.

Reply:
That's right, sorry!